animata/button/hero-button.tsx

```diff
@@ -0,0 +1,100 @@
+            </div>
+          </div>
+
+          {/* Additional arrows that appear when hovered */}
+          <div
+            className={`absolute flex items-center justify-center ${isHovered ? "opacity-100" : "opacity-0"} transition-opacity duration-300`}
+          >
+            {Array(7)
+              .fill(null)
+              .map((_, index) => (
+                <div
+                  key={index}
+                  className="relative flex h-6 w-6 items-center justify-center transition-opacity duration-300"
+                >
+                  <div className="relative h-4 w-4">
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 1} left-0 top-0`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 2} left-[3px] top-0`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 3} left-[3px] top-[3px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 4} left-[6px] top-[3px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 5} left-[6px] top-[6px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 6} left-[9px] top-[6px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 7} left-[3px] top-[9px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 8} left-[6px] top-[9px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 9} left-0 top-[12px]`}
+                    ></div>
+                    <div
+                      className={`absolute h-[2px] w-[2px] rounded-full bg-black animate-moving-arrows-item-${index + 10} left-[3px] top-[12px]`}
+                    ></div>
+                  </div>
+                </div>
+              ))}
+          </div>
+        </div>
+
+        {/* Text section on the extreme right with animation on hover */}
+        <span
+          className={`text-md ml-auto transform font-medium transition-all duration-500 ease-in-out ${isHovered ? "-translate-x-2 opacity-0" : "translate-x-0 opacity-100"}`}
+        >
```

Comment:
@NishantSinghhhhh could you make sure this button looks center aligned?

Reply:
Sure Sir